docs/networking/best-practice.md

```diff
@@ -0,0 +1,235 @@
+
+- Nics are broken
+- Nics are with insufficent hardware capacity
+- Nics are not with expected features
+
+You may follow the below steps and run them in each node step by step.
+
+### Validate the new Nics
+
+1. Confirm with SUSE/Harvester if the new Nics are certificated by the related Harvester version.
+
+1. Test the new nics in non-production environment.
+
+### Pre-check
+
+1. Check all VMs from the Harvester [VM page](../vm/access-to-the-vm.md#access-with-the-harvester-ui), they should be either running or stopped.
+
+1. Check all Longhorn volumes are healthy from the [embedded Longhorn dashboard](../troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards). It also displays those none-VM related volumes.
+
+1. (Optional) Generate a [support bundle](../troubleshooting/harvester.md#generate-a-support-bundle) for comparison with the new node later.
+
+### Collect Information
+
+Before any action is taken, it is important to collect the current network information and status.
+
+- Harvester network configuration
+
+    Harvester creates a `mgmt-bo` bond interface for the [built-in cluster network](../networking/clusternetwork.md/#built-in-cluster-network) by default, and will create a new bond interface for the up link of each new [custom cluster network](../networking/clusternetwork.md/#custom-cluster-network). Harvester saves the network related config in file `/oem/90_custom.yaml`.
+
+    Example: The `mgmt-bo` bond interface has `ens3` nic added.
+
+    ```
+    - path: /etc/sysconfig/network/ifcfg-mgmt-bo
+      permissions: 384
+      owner: 0
+      group: 0
+      content: |+
+        STARTMODE='onboot'
+        BONDING_MASTER='yes'
+        BOOTPROTO='none'
+        POST_UP_SCRIPT="wicked:setup_bond.sh"
+
+
+        BONDING_SLAVE_0='ens3'
+
+        BONDING_MODULE_OPTS='miimon=100 mode=active-backup '
+
+        DHCLIENT_SET_DEFAULT_ROUTE='no'
+
+      encoding: ""
+      ownerstring: ""
+
+    - path: /etc/sysconfig/network/ifcfg-ens3
+      permissions: 384
+      owner: 0
+      group: 0
+      content: |
+        STARTMODE='hotplug'
+        BOOTPROTO='none'
+      encoding: ""
+      ownerstring: ""
+    ```
+
+- ip link
+
+    `ip link` displays all the physical nics, their master if exists, and the `state`.
+
+    Example:
+
+    ```
+    ip link | grep master -1
+
+    2: ens3: <BROADCAST,MULTICAST,SLAVE,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast master mgmt-bo state UP mode DEFAULT group default qlen 1000
+        link/ether 52:54:00:03:3a:e4 brd ff:ff:ff:ff:ff:ff
+    --
+
+    4: mgmt-bo: <BROADCAST,MULTICAST,MASTER,UP,LOWER_UP> mtu 1500 qdisc noqueue master mgmt-br state UP mode DEFAULT group default qlen 1000
+        link/ether 52:54:00:03:3a:e4 brd ff:ff:ff:ff:ff:ff
+
+    ```
+
+- lspci
+
+    `lspci` shows all pci devices in summary, it is easy to identify the network nics.
+
+    Example:
```

Comment:
This doc, is more for IT/network adminstrators to follow to do things step by step, it is more technical, please keep original format.

Reply:
I disagree. The section title is "Collect Information" and the first item in your draft is a type of information. This is not a procedure because you used an unordered list. If you wanted users to follow steps, you should have used an ordered list. Lastly, I did not remove any technical information. I simply made it easier to understand.

You can reject the suggestion if you want.